src/js/lib/search.js

```diff
@@ -325,5 +382,67 @@ export const searchResultToLearningResource = result => ({
   topics:        result.topics ? result.topics.map(topic => ({ name: topic })) : [],
   runs:          "runs" in result ? result.runs : [],
   audience:      result.audience,
-  certification: result.certification
+  certification: result.certification,
+  content_title: result.content_title,
+  run_title:     result.run_title || null,
+  run_slug:      result.run_slug || null,
+  content_type:  result.content_type || null,
+  url:           getResultUrl(result) || null,
+  short_url:     result.short_url || null,
+  course_id:     result.course_id || null,
+  description:   result.short_description || null
 })
+
+export const getCoverImageUrl = result => {
+  if (!emptyOrNil(result.image_src)) {
+    return result.image_src
+  } else {
+    switch (result.content_type) {
+    case CONTENT_TYPE_PAGE:
+      return "/images/page_thumbnail.png"
+    case CONTENT_TYPE_PDF:
+      return "/images/pdf_thumbnail.png"
+    case CONTENT_TYPE_VIDEO:
+      return "/images/video_thumbnail.png"
+    default:
+      return "/images/default_thumbnail.png"
+    }
+  }
+}
```

Comment:
should I expect that these images will work for resources locally? Images currently work in the course search, but I ran a re-index after checking out https://github.com/mitodl/open-discussions/pull/3159 and none of the images are showing up for me on the resource search:

![Screenshot from 2020-09-28 10-18-15](https://user-images.githubusercontent.com/6207644/94444136-fc424380-0173-11eb-9e97-49b280add07b.png)

Reply:
also in this screenshot, I'm seeing the `COURSE RESOURCES` buttons aren't lining up with the left-hand edge of the result cards

Reply:
Forgot I need to get these images still, but maybe they can go in a future PR, the issue mentioned thumbnails are out of scope.  @abdulkdawson do you have web/video/generic thumbnail images for resource search available?

Reply:
The only resource thumbs that should be displaying right now are for youtube videos (content_type="video", file_type="video/youtube")

Reply:
... and pdf thumbnails should work

Reply:
Added the additional thumbnails